### ISSUE TITLE
travis failing since 1.1.1

### ISSUE THREAD

**Issue by nicolaiarocci** [pyeve/eve#1376]
See https://travis-ci.org/github/pyeve/eve/builds/670870235

I suspect a versioning issue on some dependency, or I might have messed up with latest rebases, but I do not have nor I will have time to look at it anytime soon. If anyone is willing to give it a go, please do. 

@arnauorriols @hlongmore I am pinging you both because you  have been working with the test suite lately.

**Comment by hlongmore** [comment]
The tests are all failing in the same method, in `eve/tests/config.py`
```python
    def test_custom_kwargs(self):
        self.app = Eve("unittest", static_folder="/", settings=self.settings_file)
        self.assertEqual(self.app.static_folder, "/")
```
which hasn't been changed since `ecbfe3c81d6f9fef8e6100b2f85343063e401172`. Though the formatting of the error reports are different, they all basically fail like so: 

`AssertionError: '/opt/python/2.7.15/lib/python2.7/unittest/' != '/'`

In the project I was working on that necessitated needing to fix #1338 I set eve in my `Pipfile` to use pyeve/eve.git instead of my branch since it was merged, and did a `pipenv update`. After that I started getting errors in Flask. I checked my Pipfile.lock, and saw Flask was updated from `1.1.1` to `1.1.2`. I pinned Flask to `1.1.1` and it started working again. Since this method is basically testing that `static_folder` is getting set properly, I suspect it is due to a change in Flask. I'm not going to guarantee I'll have time to look into it further for a bit (I started a new job this week), but if I do, that's where I'll start looking.
